04-store-api/starter/controllers/products.js

```diff
@@ -0,0 +1,98 @@
+const Product = require("../models/product");
+require("colors");
```

Comment:
I don't see this being used anywhere, unused import?

Reply:
i'm sorry i'm using it for debugging sometimes. it paints console logs in the terminal. i forgot to delete it